econml/sklearn_extensions/ensemble.py

```diff
@@ -462,7 +459,7 @@ def fit(self, X, y, sample_weight=None, sample_var=None):
         """
 
         # Validate or convert input data
-        X = check_array(X, accept_sparse="csc", dtype=DTYPE)
+        X = check_array(X, accept_sparse="csc", dtype=np.float32)
```

Comment:
We are using the _tree backend of sklearn. Would it be a problem if they change the native dtype in their _tree implementation. I suspect it's benign and we'll discover it.

Reply:
Yes, to some extent we have just moved the problem around somewhat; however as you say it seems like we would discover the problem if they change datatypes and it's one less internal thing that we're referencing this way.